lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -220,19 +220,22 @@ def check_alt_backend(alt_backend):
     fig.canvas.mpl_connect("close_event", print)
 
     result = io.BytesIO()
-    fig.savefig(result, format='png')
+    fig.savefig(result, format='png', dpi=100)
 
     plt.show()
 
     # Ensure that the window is really closed.
     plt.pause(0.5)
 
-    # Test that saving works after interactive window is closed, but the figure
-    # is not deleted.
+    # When the figure is closed, it's manager is removed and the canvas is reset to
```

Comment:
Maybe explicitly add a test here that type(figure.canvas) == FigureCanvasBase?  (This would catch any missing super().destroy() call.)

Reply:
Done.